src/views/Home/__tests__/Home.en.test.js

```diff
@@ -73,5 +73,16 @@ describe('Home component(English)', () => {
 		const link = container.querySelector('a');
 		fireEvent.click(link);
 		expect(analyticsHandler).toHaveBeenCalled();
+		expect(screen.getByLabelText('pager navigation')).toBeInTheDocument();
+		// Navigate to next page with next pager item button
+		await act(async () => {
```

Comment:
act is not necessary here... https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily